### PR TITLE
Made cosmetic edits to the Steering council docs.

### PR DESCRIPTION
**Pull request by sarahboyce**
[Steering Council docs](https://django--18607.org.readthedocs.build/en/18607/internals/organization.html#steering-council)

### CODE REVIEW

File: docs/internals/organization.txt [hunk 1]
```diff
@@ -168,8 +168,7 @@ Role
 The steering council is a group of experienced contributors who:
 
 - provide oversight of Django's development and release process,
-- assist in setting the direction of feature development and releases,
-- take part in filling certain roles, and
```
Comment (sarahboyce) [inline docs/internals/organization.txt:172]: I have removed "take part in filling certain roles" as I think it is too vague to add value to a role description
I can change this to something specific if required

File: docs/internals/organization.txt [hunk 2]
```diff
@@ -210,10 +209,13 @@ who demonstrate:
   This history must begin at least 18 months prior to the individual's
   candidacy for the Steering Council, and include substantive contributions in
   at least two of these bullet points:
-  - Code contributions on Django projects or major third-party packages in the Django ecosystem
+
```
Comment (sarahboyce) [inline docs/internals/organization.txt:212]: This missing blank line meant the sub-bullet was not formatted correctly (see [first bullet](https://docs.djangoproject.com/en/5.1/internals/organization/#id7))

File: docs/internals/organization.txt (hunk 1 above)
Comment (sarahboyce) [reply docs/internals/organization.txt]: Although I don't think this is clear, this is quoted directly from [DEP-10](https://github.com/django/deps/blob/main/final/0010-new-governance.rst#repurposed-role-technical-board). So I will leave this for now as this should be aligned

Comment (sarahboyce) [reply docs/internals/organization.txt]: https://github.com/django/deps/pull/93

**Review by LilyFoote — APPROVED** [review]

File: docs/internals/organization.txt (hunk 1 above)
Comment (sarahboyce) [reply docs/internals/organization.txt:172]: Updated the DEP and now aligned this here

**Review by LilyFoote — APPROVED** [review]

**Pull request merged**